.gitlab/build_corona.yml

```diff
@@ -39,8 +39,9 @@ corona_release:
 .build_on_corona:
   stage: build
   variables:
-    ALLOC_COMMAND: "srun -p pbatch -t 30 -N 1 ${ASSIGN_ID} --interactive"
+    ALLOC_COMMAND: "srun -p pbatch -t 10 -N 1 ${ASSIGN_ID} --interactive"
```

Comment:
not sure why this was 30 because the salloc was 10.. this doesnt really change anything.